### DISCUSSION TITLE
Feedback for Simple Input Validation lesson

### DISCUSSION THREAD

**Discussion by jcollard** [CaptainCoderOrg/IntroToCSharpSite#185] [Page Feedback]
Feedback for If Statements lesson: https://csharp-develop.captaincoder.org/lessons/branching-logic/simple-input-validation

**Comment by cloudthepain** [comment]
Typo:
Before starting this lesson, you should be comfortable **using if and else statements to create decision trees in your programs**. If you need a refresher, be sure to check out the If Statements lesson.
>>> using if and else statements and creating decision trees...

Looks odd:
If this is the first time you've seen a "Try" method, it might look a little odd. Specifically, the second argument: **out result.** 
>>> Since it is kind of split up.

Explain
Note: The condition can be further "simplified" using the not operator (!):
>>> I would put a line in here that explains what the indicator is saying. Even though it is explained in another lesson, it would be better to say something like:
"This is saying if the condition is not true then..."
"If you want to learn more about this, check out..."
